DiamondLightSource/hyperion
# Ophyd pin-tip centring

Requires https://github.com/DiamondLightSource/dodal/pull/289

Provide support for running both Ophyd + AD MXSC pin tip centring workflows in hyperion.

To test:
* Confirm using the ophyd device is externally selectable
* Confirm the issues in https://github.com/DiamondLightSource/hyperion/pull/938#issuecomment-1777284170 are addressed:
    * https://github.com/DiamondLightSource/dodal/issues/290
    * https://github.com/DiamondLightSource/hyperion/issues/1068
    * https://github.com/DiamondLightSource/hyperion/issues/1069
* Confirm tests pass
